mne/viz/evoked.py

```diff
@@ -126,6 +127,36 @@ def _rgb(x, y, z):
     return np.asarray([x, y, z]).T
 
 
+def _plot_legend(pos=None, colors=None, axis=None, outlines='head'):
```

Comment:
IMO `outlines='skirt'` looks better than `'head'`, but elsewhere, we give the option of selecting either.

Not sure about the API, but I'd like to keep the option available at least here.